### PR TITLE
Add H3-OPTION variable to the local.conf

### PR DESCRIPTION
**Pull request by iartemenko**
Depending on H3 board version automatically add
U-boot/IPL option for H3 (SoC: r8a7795).

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston [hunk 1]
```diff
@@ -384,14 +384,6 @@ DISTRO_FEATURES_append = " ivi-shell"
...
 MACHINE_FEATURES_append = " cas"
 
-# U-boot/IPL option for H3 (SoC: r8a7795)
-# For H3 SiP DDR 4GiB (1GiB x 4ch)
-#H3_OPTION = "0"
-# For H3 SiP DDR 8GiB (2GiB x 4ch)
-H3_OPTION = "1"
-# For H3 SiP DDR 4GiB (2GiB x 2ch)
-#H3_OPTION = "2"
-
```
Comment (aanisov) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: please remove this with a proper reverting the hack commit

File: recipes-domd/domd-image-weston/domd-image-weston.bbappend [hunk 2]
```diff
@@ -114,6 +114,23 @@ configure_versions_rcar() {
...
+    #H3_OPTION = "1"
+    # For H3 SiP DDR 4GiB (2GiB x 2ch)
+    #H3_OPTION = "2"
+    if [ "${MACHINE}" == "salvator-xs-h3" ] || [ "${MACHINE}" == "salvator-x-h3" ];then
+        base_set_conf_value ${local_conf} H3_OPTION "0"
+    fi
+    if [ "${MACHINE}" == "salvator-xs-h3-4x2g" ] || [ "${MACHINE}" == "salvator-x-h3-4x2g" ];then
+        base_set_conf_value ${local_conf} H3_OPTION "1"
+    fi
+    if [ "${MACHINE}" == "salvator-xs-h3-2x2g" ];then
```
Comment (aanisov) [inline recipes-domd/domd-image-weston/domd-image-weston.bbappend]: I would suggest adding here machine "salvator-x-h3-2x2g" as well

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston (hunk 1 above)
Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: Done

Comment (aanisov) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston]: You missed a signed-off in a revert commit.

**Comment by aanisov** [comment]
Also, this should be tested for all available platforms before releasing

**Comment by aanisov** [comment]
Another question is if we should merge it to master now, or to salvator-xs dev branch. If to the master, than salvator-xs branch should be rebased.

**Pull request merged**